lib/beaker/subcommand.rb

```diff
@@ -53,6 +53,7 @@ def initialize(*args)
     class_option :'xml-time-order', :type => :boolean, :group => 'Beaker run'
     class_option :'debug-errors', :type => :boolean, :group => 'Beaker run'
     class_option :'exec_manual_tests', :type => :boolean, :group => 'Beaker run'
+    class_option :'test-tag-exclude', :type => :string, :group => 'Beaker run'
```

Comment:
Does it make sense to restrict this option to `exec` only? You could add it specific to that subcommand using the `method_option`...method. There's an example of it for hosts in the `init` function.

Reply:
I don't think so from the way we're invoking beaker when running subcommands. Additional options are passed into all subcommands regardless, so we'll need all subcommands to be able to parse the flag

Reply:
While we are in this section of code, should we update it to also accept `--test-tag-and` and `--test-tag-or`? Other than that, I'm a 👍 here.

Reply:
I'm happy to add those in. I wasn't sure how much change I should introduce, but I'll go update this now